packages/expo-location/src/Location.ts

```diff
@@ -269,7 +259,7 @@ export async function requestPermissionsAsync(): Promise<LocationPermissionRespo
 // @needsAudit
 /**
  * Checks user's permissions for accessing location while the app is in the foreground.
- * @return A promise that fulfills with an object of type [PermissionResponse](#permissionresponse).
+ * @return A promise that fulfills with an object of type [LocationPermissionResponse](#locationpermissionresponse).
```

Comment:
```suggestion
 * @return A promise that fulfills with an object of type [`LocationPermissionResponse`](#locationpermissionresponse).
```

Let's add the missing inline code highlight.